packages/luna/src/components/PlanCard/PlanCard.tsx

```diff
@@ -0,0 +1,206 @@
+    <div
+      className={cn(
+        "tw-relative tw-p-8 tw-border tw-shadow-plan-card tw-rounded-3xl",
+        primary
+          ? "tw-bg-brand-700 tw-border-brand-800"
+          : "tw-bg-natural-50 tw-border-natural-200"
+      )}
+    >
+      {label ? (
+        <Typography
+          element="body"
+          weight="semibold"
+          className={cn(
+            "tw-absolute -tw-top-0 tw-left-1/2 -tw-translate-x-1/2 -tw-translate-y-full",
+            "tw-border-t tw-border-r tw-border-l tw-border-natural-300 tw-rounded-t-2xl tw-p-[10px] tw-shadow-plan-card-label -tw-z-10",
+            { "tw-bg-natural-100 tw-text-brand-700": label === "most-common" },
+            { "tw-bg-brand-700 tw-text-natural-50": label === "most-valuable" }
+          )}
+        >
+          {label === "most-common" ? intl("plan.label.most-common") : null}
+          {label === "most-valuable" ? intl("plan.label.most-valuable") : null}
+        </Typography>
+      ) : null}
+      <div className="tw-flex tw-items-center tw-gap-4">
+        <Typography
+          element="subtitle-2"
+          weight="bold"
+          className={cn(
+            "tw-mb-2",
+            primary ? "tw-text-natural-50" : "tw-text-natural-950"
+          )}
+        >
+          {title}
+        </Typography>
+        {discount ? (
+          <div className="tw-self-stretch">
+            <Typography
+              element="tiny-text"
+              weight="regular"
+              className={cn(
+                "tw-text-brand-700 tw-py-[2px] tw-px-3 tw-rounded-[30px]",
+                primary ? "tw-bg-discount-primary" : "tw-bg-discount-default"
+              )}
+            >
+              {intl("plan.discount", { value: formatNumber(discount) })}
+            </Typography>
+          </div>
+        ) : null}
+      </div>
+      <Typography
+        element="tiny-text"
+        weight="semibold"
+        className={cn(
+          "tw-inline-block tw-mb-4",
+          primary ? "tw-text-natural-100" : "tw-text-natural-700"
+        )}
+      >
+        {description}
+      </Typography>
+      <Typography
+        element="h4"
+        weight="bold"
+        className={cn(
+          "tw-mb-2",
+          primary ? "tw-text-natural-50" : "tw-text-brand-700"
+        )}
+      >
+        {intl("plan.weekly-minutes", { value: formatNumber(weeklyMinutes) })}
+      </Typography>
+      <div className="tw-flex tw-gap-2 tw-items-end tw-flex-wrap">
+        <Typography
+          element="subtitle-1"
+          weight="bold"
+          className={cn(primary ? "tw-text-natural-50" : "tw-text-natural-950")}
+        >
+          {formatNumber(priceAfterDiscount, {
+            maximumFractionDigits: 2,
+          })}
+        </Typography>
+        <Typography
+          element="subtitle-1"
+          weight="bold"
+          className={cn(primary ? "tw-text-natural-50" : "tw-text-natural-950")}
+        >
+          {intl("labels.currency.egp")}
+        </Typography>
```

Comment:
Whey we are rendering this as two seperate typographies. Please compaine them into one. and use intl to concat the strings into sentence.

Reply:
i already did that at first but when I tried to add huge numbers, I needed this section to be formed of small components as possibly as I could so I could wrap it